tensorflow_datasets/core/registered.py

```diff
@@ -106,6 +107,14 @@
     version=r"[0-9]+\.[0-9]+\.[0-9]+",
 ))
 
+_skip_reg = False
+
+@contextmanager
+def skip_registration():
```

Comment:
Could you also add a test in `registered_test.py` to test this feature directly (creates a new Builder class inside the context manager, without import) ?

Reply:
Sure